src/main/java/com/quantumsoft/qupathcloud/converter/dicomizer/Wsi2dcmLibrary.java

```diff
@@ -0,0 +1,32 @@
+package com.quantumsoft.qupathcloud.converter.dicomizer;
+import com.sun.jna.Library;
+import com.sun.jna.Native;
+import com.sun.jna.NativeLibrary;
+import com.sun.jna.NativeLong;
+import com.sun.jna.Pointer;
+import com.sun.jna.ptr.DoubleByReference;
+import java.nio.ByteBuffer;
+import java.nio.DoubleBuffer;
+/**
+ * JNA Wrapper for library <b>wsi2dcm</b><br>
```

Comment:
Do we need to bring in another dependency for this? This file looks simple enough that we could go without it. A bunch of the links in this comment aren't valid and the credits and license page lists a bunch of other licenses that would be simpler to avoid reviewing if we can.

Reply:
We use JNA to simplify loading platform-specific binary of wsi2dcm from jar. This way we just need to put it into correct resource folder and that's it.

Reply:
Ok but you also seem to be using some code generation tool "JNAerator" based on the class comment here that has a bunch of dead links in it. Can we stop using that code generation tool?

In terms of using JNA OK I need to verify this is an OK dependency to link because it has an LGPL license. Also I'm not seeing any changes to pom.xml to add a dependency on JNA how is it being linked in?

Reply:
Based on https://github.com/nativelibs4java/JNAerator is LGPLv3.
Initially we used simple 'extern C' declaration on c side instead of native wrapper, so returning to that is always an option. Can't get exceptions from c code to java this way (just exit code), otherwise it would work.
We tried generic JNI approach, but headers generated this way lead to hard to diagnose crashes, which is why we switched to JNAerator. Egor Lezhin could probably take another look at generating JNI headers without JNAerator.

https://github.com/java-native-access/jna is "licensed under the LGPL, version 2.1 or later, and (from version 4.0 onward) the Apache Software License, version 2.0. Commercial license arrangements are negotiable."
We use 5.0 currently.
No pom change because I forgot to remove it when disabling the old dicomizer. So it's already there.
<dependency>
			<groupId>net.java.dev.jna</groupId>
			<artifactId>jna</artifactId>
			<version>5.0.0</version>
</dependency>

Reply:
Ok let me double check on both of these items and get back to you.

Reply:
Ok confirmed that added these dependencies is OK.